site/content/docs/5.3/utilities/colors.md

```diff
@@ -148,14 +148,15 @@ When used in a dark variant context, `--bs-primary-text-rgb` will use the value
 
 To change that opacity, override `--bs-text-opacity` via custom styles or inline styles.
 
-{{< example >}}
+<!-- Boosted mod: exclude-from-pa11y-analysis because the use of `.text-primary` on small texts + opacity -->
```

Comment:
Could/Sgould we add some context here?

`...because the use of `.text-primary` on small texts + opacity` leads to insufficient color contrast / failed to reach minimum required ratio of ...?